### DISCUSSION TITLE
App is Not Opening

### DISCUSSION THREAD

**Discussion by callmeisaacyt** [lllyasviel/Fooocus#3698] [Q&A]
i Have downloaded the 2.5 and afte extract. i choose the run realistic.bat . and afte dowload the 6 gb file it said press any key. and after i press, the app doesnt open for me. What did i do wrong ?

**Comment by Pixel-47** [comment]
Ja ją 1.5 h nie moge wypakować 7-zip jaki i Windowsowym programem do otwierania, Masakra.
